modules/@apostrophecms/attachment/index.js

```diff
@@ -51,8 +51,9 @@ module.exports = {
         extensions: [
           'gif',
           'jpg',
-          'png'
-        ].concat(self.options.svgImages ? [ 'svg' ] : []),
+          'png',
+          'svg'
```

Comment:
Not having an option is bold, but let's try it man, see if anybody has real concerns; can always add options, can't take 'em away